### ISSUE TITLE
On Congratulating user for Day 1 or Day 100, not checking clearly whether it's Day 1 or Day 100

### ISSUE THREAD

**Issue by amandeepmittal** [FreeCodeCamp/100DaysOfCode-twitter-bot#15]
Check the condition if it's `Day 1` or `Day 100` to decide whether its first day or last day.
Also check the condition and exclude `Day 10`,  no tweet should be sent on `Day 10`.

**Comment by QuincyLarson** [comment]
@amandeepmittal yes - I noticed this bug when I was looking at the PR. We may be able to use a regex to check for a non-digit character after the 1.

**Comment by amandeepmittal** [comment]
@QuincyLarson yes, @stephanie56 has done something similar. I'll check it first.
